user_class/user.rb

```diff
@@ -1,2 +1,105 @@
 class User
+  @@VALID_ATR = %i(first_name last_name email age address)
```

Comment:
Why is this a global variable? Wouldn't be better a constant in the class or do you need it somewhere else?